test/fixtures/rules/std_rule_cases/L003.yml

```
@@ -1100,3 +1097,54 @@ test_tsql_bubble_up_newline_after_fix:
   configs:
     core:
       dialect: tsql
+
+test_tsql_cross_apply_indentation:
+  # Test for behavior in issue #3672
+  pass_str: |
+    SELECT
+      table1.col,
+      table2.col
+    FROM table1
+    CROSS APPLY (
+      VALUES ((1), (2))
+    ) AS table2(col)
+    INNER JOIN table3
+      ON table1.col = table3.col;
```

The default spacing is 4 spaces so you need to use that in all your test cases:

```suggestion
    SELECT
        table1.col,
        table2.col
    FROM table1
    CROSS APPLY (
        VALUES ((1), (2))
    ) AS table2(col)
    INNER JOIN table3
        ON table1.col = table3.col;
```